src/lib/components/market/response_render_results_status.ts

```diff
@@ -0,0 +1,155 @@
+function overrideOnResponseRenderResults(proto: any) {
```

Comment:
These functions can be included as part of the inner impl of the component, specifically when it starts you can handle getting the proto reference and overriding. Since at the end of the day, it is tightly coupled anyways and isn't capable of operating outside of a global reference to that component.